editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -1210,7 +1210,16 @@ void EditorAssetLibrary::_http_request_completed(int p_status, int p_code, const
 			library_vb->add_child(asset_bottom_page);
 
 			if (result.empty()) {
-				library_error->set_text(vformat(TTR("No results for \"%s\"."), filter->get_text()));
+				if (filter->get_text() != String()) {
+					library_error->set_text(
+							vformat(TTR("No results for \"%s\" compatible with Godot %s."), filter->get_text(), String(VERSION_BRANCH)));
```

Comment:
I'm not sure I would put the version number here, I fear it might be jarring to users. They might wonder if they should change something to get access to the compatible results.

It's good to have in the "no content at all" case though.